### PR TITLE
Added dns provider for ipv64.net

### PR DESCRIPTION
**Pull request by romanlum**
This PR adds support for the dns api of ipv64.net

### CODE REVIEW

**Comment by romanlum** [comment]
Added issue for support
https://github.com/acmesh-official/acme.sh/issues/4419

**Comment by romanlum** [comment]
Pipeline added.
The docker workflow fails although the certificates are created correctly.
Can you have a look if there is a problem with your workflow?

**Review by Zoey2936 — APPROVED** [review]

**Comment by romanlum** [comment]
Thanks for approving the pull request.
Is anything more todo before merging the PR?

When should i create the WIKI article for ipv64.net dns api?

**Comment by Bergruebe** [comment]
It would be great, if this PR could be merged.
Can I help anywhere?

**Comment by Neilpang** [comment]
The dns api test must be passing.

https://github.com/romanlum/acme.sh/actions/runs/3581360113

**Comment by romanlum** [comment]
I updated the branch to match the current dev and fixed the dns api tests
Now all pipelines are green :)

https://github.com/romanlum/acme.sh/actions/runs/4053033990
https://github.com/romanlum/acme.sh/actions/runs/4053033971
https://github.com/romanlum/acme.sh/actions/runs/4053033969

**Comment by romanlum** [comment]
@Neilpang 
Would be great if you could merge the pull request :)
Thanks

**Comment by Neilpang** [comment]
can you please add the usage here:  https://github.com/acmesh-official/acme.sh/wiki/dnsapi2

File: dnsapi/dns_ipv64.sh [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  if ! _get_root "$fulldomain"; then
+    _err "invalid domain" "$fulldomain"
+    return 1
+  fi
+  _debug _sub_domain "$_sub_domain"
+  _debug _domain "$_domain"
+
+  # convert to lower case
+  _domain="$(echo "$_domain" | tr '[:upper:]' '[:lower:]')"
```
Comment (Neilpang) [inline dnsapi/dns_ipv64.sh:40]: use the function "_upper_case()" and "_lower_case"

**Comment by romanlum** [comment]
> can you please add the usage here: https://github.com/acmesh-official/acme.sh/wiki/dnsapi2

How can I add this. I am not able to edit the page.

File: dnsapi/dns_ipv64.sh (hunk 1 above)
Comment (romanlum) [reply dnsapi/dns_ipv64.sh]: Changed code to use the lower_case function

**Comment by romanlum** [comment]
Code changed all pipelines are green :)
https://github.com/romanlum/acme.sh/actions/runs/4269752746
https://github.com/romanlum/acme.sh/actions/runs/4269752743
https://github.com/romanlum/acme.sh/actions/runs/4269752741

**Comment by Neilpang** [comment]
>> How can I add this. I am not able to edit the page.

You can edit it now

**Comment by romanlum** [comment]
updated https://github.com/acmesh-official/acme.sh/wiki/dnsapi2 and added ipv64 as provider 155

**Pull request merged**